include/seastar/http/httpd.hh

```diff
@@ -100,18 +100,6 @@ public:
 
     future<> start_response();
 
-    static short hex_to_byte(char c);
-
-    /**
-     * Convert a hex encoded 2 bytes substring to char
-     */
-    static char hexstr_to_char(const std::string_view& in, size_t from);
-
-    /**
-     * URL_decode a substring and place it in the given out sstring
-     */
-    static bool url_decode(const std::string_view& in, sstring& out);
```

Comment:
> Very good. Please add a working program under demos/. It can use threads or coroutines so it's nice.

OK

> Does the client support Connection: keep-alive?

Client doesn't care, as far as I can tell. It grabs the socket and can be used to call `make_request()` several times in a row. It's up to the caller whether to trow away an existing client, or to re-use it